packages/dds/tree/.vscode/Tree.code-workspace

```diff
@@ -10,6 +10,10 @@
 		}
 	],
 	"settings": {
-		"search.followSymlinks": false
+		"search.followSymlinks": false,
+		"typescript.preferences.autoImportFileExcludePatterns": [
```

Comment:
seems like a good thing to include in the root workspace as well (.vscode/settings.json). If you don't want to do that here, I can make a separate PR for it. let me know :)

Reply:
I'm going to try and land this as is to unblock the perf tests pipeline. Moving this to the folder settings (no need for it to be in both), possible along with `"typescript.preferences.preferTypeOnlyAutoImports": true` seems like a good idea if they work well for all the packages.

Reply:
sounds good to me